src/main/resources/hudson/plugins/git/util/BuildDetails/index_it.properties

```diff
@@ -0,0 +1,2 @@
+Git\ Build\ Details=Dati del progetto git
+Revision=Revisione
```

Comment:
Best to leave such things to translators. (Anyway the movement is towards having translations in separate repositories.)